conf/broker.conf

```diff
@@ -1232,6 +1232,9 @@ exposeManagedCursorMetricsInPrometheus=false
 # Set it to 0 to disable timeout.
 metricsServletTimeoutMs=30000
 
+# Enable or disable broker bundles metrics. The default value is false.
+exposeBunlesMetricsInPrometheus=false
```

Comment:
```suggestion
exposeBundlesMetricsInPrometheus=false
```

Reply:
We don't have a release that contains https://github.com/apache/pulsar/pull/12366, so you can fix the typo directly.